torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1272,13 +1272,13 @@ def method_tests():
         ('kthvalue', (), (1, 0, True), 'scalar_keepdim_dim', (), [1]),
         ('quantile', (S, S, S), (0.5,)),
         ('quantile', (S, S, S), (0.5, 0), 'dim', (), [1]),
-        ('quantile', (S, S, S), (0.5, None, True), 'keepdim'),
-        ('quantile', (S, S, S), (0.5, 0, True), 'keepdim_dim', (), [1]),
+        ('quantile', (S, S, S), (0.5, None, 'linear', True), 'keepdim'),
```

Comment:
In the future we should produce a ReductionInfo class from the OpInfo base class and move quantile and nanquantile to it.

Reply:
Created issue here https://github.com/pytorch/pytorch/issues/49746